distribution/rpm/vector.spec

```diff
@@ -31,7 +31,7 @@ Name: %{_name}
 Summary: A lightweight and ultra-fast tool for building observability pipelines
 Version: %{_cleaned_version}
 Release: %{_release}
-License: ASL 2.0
+License: MPL-2.0
```

Comment:
🤦

Reply:
(We didn't update our nix one either, but I just deleted that one). I need to PR or open an issue for the community nix package, which also lists us as ASL2.0